src/cryptoadvance/specter/templates/base.jinja

```diff
@@ -38,6 +38,9 @@
 				border-right: 0;
 				{% endif %}
 			}
+			tr:hover {
+				background: rgba(255,255,255,0.03);
```

Comment:
This has introduced the hovering effect back to the balance display.

Reply:
Yeah, too general in` base.jinja`, indeed. Will add a hover-css-class for the respective rows.